src/common/components/card/LargeCollectionCard.tsx

```diff
@@ -16,7 +17,10 @@ function LargeCollectionCard({
     <Card id={id} className={styles.largeCollectionCard}>
       <Card.Content className={styles.text}>
         <Card.Title as="h3" variant="h2" title={title} href={href} />
-        <Card.InfoLines variant="body-l" infoLines={infoLines} />
+        <Card.ClampedCardInfoLines
+          variant="body-l"
+          infoLines={formatToSingleTag(infoLines)}
```

Comment:
With this implementation, wouldn't the resulting output be something like `[<undefined>lines</undefined>]`?